pyEpiabm/pyEpiabm/intervention/social_distancing.py

```diff
@@ -0,0 +1,60 @@
+#
+# Social distancing Class
+#
+
+import random
+
+from pyEpiabm.core import Parameters
+from pyEpiabm.intervention import AbstractIntervention
+
+
+class SocialDistancing(AbstractIntervention):
+    """Social distancing intervention
+    Social distancing is based on the number of infecious persons in their
+    microcells and stop distancing people after their distancing period.
+    """
+
+    def __init__(
+        self,
+        distancing_duration,
+        distancing_delay,
+        case_microcell_threshold,
+        distancing_compliant,
+        distancing_enhanced_prob,
+        population,
+        **kwargs
+    ):
+        self.distancing_duration = distancing_duration
+        self.distancing_delay = distancing_delay
+        self.case_microcell_threshold = case_microcell_threshold
+        self.distancing_compliant = distancing_compliant
+        self.distancing_enhanced_prob = distancing_enhanced_prob
+        super(SocialDistancing, self).__init__(population=population,
+                                               **kwargs)
+
+    def __call__(self, time):
```

Comment:
I really don't understand what's going on with the age dependence here - if you have explained in the wiki, could you put a link in the docstring for this function (which you should also write!)

Reply:
I wrote a brief explanation in the docstring and also the put the wiki page link there (wiki hasn't finished yet).